virtwho/logger.py

```diff
@@ -1,3 +1,57 @@
-"""
-Define the logger handler for project
-"""
+import os
+import time
+import logging
+
+
+class Logger:
+    """
+    Usage:
+        from hypervisor import log
+        logger = log.getLogger(__name__)
+        logger.info("abc")
+        logger.debug("abc")
+        logger.error("abc")
+        logger.warning("abc")
+    """
+
+    def __init__(self, logger=None):
+        """
+        The log message will output to file and console.
+        Define the log path, log file, log level, log formatter.
+        The default log directory is logs.
+        """
+        self.logger = logging.getLogger(logger)
+        self.logger.setLevel(logging.DEBUG)
+        self.logger.handlers = []
+        self.log_path = os.path.join(
+            os.path.dirname(os.path.dirname(os.path.realpath(__file__))),
+            'logs')
+        if not os.path.exists(self.log_path):os.mkdir(self.log_path)
```

Comment:
It's my fault, it's not pythonic, please change it to:
```
if not os.path.exists(self.log_path):
    os.mkdir(self.log_path)
```

Reply:
Done